tests/manage/test_run_io_multiple_pods.py

```diff
@@ -0,0 +1,103 @@
+import pytest
+import logging
+import random
+from concurrent.futures import ThreadPoolExecutor
+from ocs_ci.ocs import constants
+
+from ocs_ci.framework.testlib import ManageTest, tier2
+from tests.fixtures import (
+    create_rbd_storageclass, create_ceph_block_pool,
+    create_rbd_secret, create_pods, create_pvcs, create_project,
+    create_cephfs_secret, create_cephfs_storageclass
+)
+
+
+logger = logging.getLogger(__name__)
+
+
+@pytest.fixture()
+def init_pvc_size(request):
+    """
+    Initialize the PVC size for PVC creation
+    """
+    class_instance = request.node.cls
+    class_instance.pvc_size_int = getattr(
+        class_instance, 'pvc_size_int', random.randint(1, 10)
+    )
+    class_instance.pvc_size = f'{class_instance.pvc_size_int}Gi'
+
+
+@tier2
+@pytest.mark.usefixtures(
+    create_project.__name__,
+    init_pvc_size.__name__,
+)
+class BaseRunIOMultiplePods(ManageTest):
+    """
+    Run IO on multiple pods in parallel
+    """
+    pvc_size_int = 5
+    num_of_pvcs = 10
+    interface = None
+
+    def run_io_multiple_pods(self):
+        """
+        Run IO on multiple pods in parallel
+        """
+        results = list()
+        with ThreadPoolExecutor(max_workers=self.num_of_pvcs) as executor:
```

Comment:
@ebenahar 
I don't think we need a ThreadPoolexecutor here, because pod.run_io itself is async and threaded which spawns a thread to perform task.
``` python
for pod in self.pod_objs:
    pod.run_io('fs', f'{self.pv_size_int -1}G')
``` 
should be enough

Reply:
thanks, done